<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManagerDialog</class>
 <widget class="QDialog" name="DeviceManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>263</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>263</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设备管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>设备选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboDeviceType">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>575</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>索引</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboDeviceIndex">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>166</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labChannel">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="text">
           <string>通道</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboChannelIndex">
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="title">
      <string>设备操作</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QtMaterialRaisedButton" name="btnCloseDevice">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtMaterialRaisedButton" name="btnDeviceInfo">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>设备信息</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QtMaterialRaisedButton" name="btnStartDevice">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>启动</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtMaterialRaisedButton" name="btnStopDevice">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QtMaterialRaisedButton" name="btnOpenDevice">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="text">
         <string>打开设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header>qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
